src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/DocumentToRemove.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class DocumentToRemove {
+    private String documentToRemoveUrl;
```

Comment:
Is it possible to reuse CaseDocument here rather than have these new fields?

Reply:
Good point will look to use this model